plugins/auth-backend-module-oidc-provider/src/authenticator.ts

```diff
@@ -84,6 +86,9 @@ export const oidcAuthenticator = createOAuthAuthenticator({
         'The oidc provider no longer supports the "scope" configuration option. Please use the "additionalScopes" option instead.',
       );
     }
+    const httpOptionsProvider = createHttpOptionsProvider({
+      timeout: config.getOptionalNumber('timeout'),
```

Comment:
you need to add this to `config.d.ts` also

Reply:
also it would be good if this would be of type `HumanDuration` as in other places in Backstage

Reply:
Hi @deejay1, thanks for your feedback. I have changed the code. Please have a look 🙂